resources/dicts/relationship_events/normal_interactions/comfortable/increase.json

```
@@ -136,7 +149,9 @@
 		"id": "comfortable_kit_inc_med2",
 		"interactions": [
 			"m_c wants to snuggle with r_c.",
-			"m_c is hiding under a bush from r_c, but {PRONOUN/m_c/subject} can't stop giggling."
+			"m_c is hiding under a bush from r_c, but {PRONOUN/m_c/subject} can't stop giggling.",
+			"m_c plops down near r_c, sneaking glances at {PRONOUN/r_c/object} expectantly. r_c doesn't understand until m_c wonders aloud to {PRONOUN/m_c/self} if r_c would play with {PRONOUN/m_c/object} if {PRONOUN/m_c/subject} asked.",
```

```suggestion
			"m_c plops down near r_c, sneaking glances at {PRONOUN/r_c/object} expectantly. r_c doesn't understand until m_c muses aloud to {PRONOUN/m_c/self} about whether or not r_c would play with {PRONOUN/m_c/object} if {PRONOUN/m_c/subject} asked.",
```
reducing some repetition, and "muses" makes the intention a bit clearer i think